Production/govuk_ios/ViewControllers/SettingsViewController.swift

```diff
@@ -21,5 +23,66 @@ class SettingsViewController: BaseViewController,
         super.viewDidLoad()
 
         view.backgroundColor = UIColor.govUK.fills.surfaceBackground
+
+        title = viewModel.title
+
+        navigationController?.setNavigationBarHidden(false, animated: false)
+        navigationController?.navigationBar.prefersLargeTitles = true
+
+        scrollview.translatesAutoresizingMaskIntoConstraints = false
+        view.addSubview(scrollview)
+
+        addContent()
+
+        constraints()
+    }
+
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+
+        let screen = AnalyticsScreen(
```

Comment:
Is there a reason that the pattern used in `HomeViewController` i.e making the view controller conform to `TrackableScreen` wasn't suitable here?
Also, I think the tracking tag doesn't match the existing pattern as its not all lowercase.
If we stick with this pattern, I would suggest having title and trackingName as separate values in the view model as there is a chance one may change and the other not change. At very least we should have a unit test to validate the tracking title separately so we know if we break it.